components/timeline/Timeline.tsx

```diff
@@ -19,30 +20,50 @@ export default class Timeline extends React.Component<TimelineProps, any> {
   };
 
   render() {
-    const { prefixCls, children, pending, pendingDot, className, ...restProps } = this.props;
+    const {
+      prefixCls,
+      pending = null, pendingDot = <Icon type="loading" />,
+      children, className, reverse = false,
+      ...ulProps,
```

Comment:
@zheeeng 
1. `ulProps` => `restProps`.
2. `pending` and `pendingDot` should not be assigned here, just keep the original code.
3. Give `reverse` a default value in `defaultProps`.